### ISSUE TITLE
The poor performances of the multi-labels BraTs dataset

### ISSUE THREAD

**Issue by smallboy-code** [WuJunde/MedSegDiff#138]
The raw images:

![T1](https://github.com/WuJunde/MedSegDiff/assets/61675340/8b6ed862-1a3b-4130-9733-05c1b9f207d9)
![visdom_image (3)](https://github.com/WuJunde/MedSegDiff/assets/61675340/011ada15-2fb6-4270-8737-5786c90422ca)
![visdom_image (2)](https://github.com/WuJunde/MedSegDiff/assets/61675340/01125e40-87dc-4fdb-b4d7-2a9753b4a501)
![visdom_image (1)](https://github.com/WuJunde/MedSegDiff/assets/61675340/279d2abf-7ded-4f40-8629-920e69c13c65)

The Ground Truth:
![图片1](https://github.com/WuJunde/MedSegDiff/assets/61675340/c9fab943-1617-43d5-a0e2-b6d327f385e9)

The output sample of 5 ensembles:

![图片3](https://github.com/WuJunde/MedSegDiff/assets/61675340/078d3584-2af3-4ac2-87ea-fe2bd27ca869)

So why these outputs contain the brain boundary? And how to set 0,1, 2, 4?

**Comment by princerice** [comment]
I would like to know your batch and the number of training steps

**Comment by smallboy-code** [comment]
batch_size: 32  and training steps : 30000

**Comment by princerice** [comment]
Thank you for your answer. My batch is set to 8 and I use A5000GPU. I would like to know the size of the GPU memory you use

**Comment by smallboy-code** [comment]
I using the four A100 GPUs. And I get the normal mask without brain tissue boundary, but the Dice is lower, about 0.4. So do you have this issue?

**Comment by princerice** [comment]
I trained 40,000 steps on a gpu and the results are not good and we may need to adjust and improve ourselves

**Comment by princerice** [comment]
I'm also confused that the loss has leveled off but the results are far from satisfactory

**Comment by smallboy-code** [comment]
Yes, and I want to know the clamp range in the final "process_xstart" function. I am use (0,3).

**Comment by princerice** [comment]
This part is not clear to me yet, I am just trying to reproduce and learn the code

**Comment by princerice** [comment]
> Yes, and I want to know the clamp range in the final "process_xstart" function. I am use (0,3).
Does process_xstart mean the start t position of the diffusion model? I don't think it matters much if it's in the 0-3 range

**Comment by teapanda628** [comment]
Hi！I have trouble with multi-label segmentation. Could you tell me if I want to have a triple classes segmentation (background+label 1+label2), where and which parameters I need to change
